tests/python/test_unsat_core.py

```diff
@@ -18,7 +18,7 @@
 import smt_switch as ss
 
 # z3 backend doesn't support unsat assumptions yet
```

Comment:
Can this be deleted?

Reply:
Yep, good catch! Fixed with b49f50d01f6a3b5616e8f6cb23c390d45fddab2b.